packages/salesforcedx-lwc-language-server/typings/glob-to-regexp.d.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+declare module 'glob-to-regexp' {
```

Comment:
a new added library that doesn't have TS support is a bad sign.  I looked after seeing its lack of TS, and it's last published 7 years ago, archived, not got a valid license (probably not ok to use)

I think this library needs to go and use somethign else.